tests/test_chatapproach.py

```diff
@@ -16,6 +16,7 @@ def chat_approach():
         chatgpt_deployment="chat",
         embedding_deployment="embeddings",
         embedding_model="text-",
+        embedding_dimensions=1536,
```

Comment:
Can you make 1536 a constant somewhere and reuse it in these tests for when we need to update that